com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -80,10 +95,19 @@ public enum LightRenderingMode
         PerPixel = 1,
     }
 
+    /// <summary>
+    /// Controled if profiling should be logged or not. This enum is not longer in use.
+    /// </summary>
```

Comment:
In theory the docs API tool should ignore obsolete API. If it's not then it's ok, but we should figure out why the tool is not ignoring this.